app/models/provider.rb

```diff
@@ -40,7 +40,20 @@ class Provider < ApplicationRecord
   has_and_belongs_to_many :organisations, join_table: :organisation_provider
 
   has_many :sites
-  has_many :enrichments, foreign_key: :provider_code, primary_key: :provider_code, class_name: "ProviderEnrichment"
+  has_many :enrichments,
+           foreign_key: :provider_code,
+           primary_key: :provider_code,
+           class_name: "ProviderEnrichment"
+
+  scope :changed_since, ->(datetime) do
```

Comment:
correctiveness of `change_since`

so provider_enrichment is it `published` or `draft`?

if its `draft` is it intended  to state the provider has changed? given the aspect that of `publish`

Reply:
You're right @defong, we should only be picking up published changes. Good catch!

Reply:
@defong fixed, we hope.